### ISSUE TITLE
[Request] Changing date without reset

### ISSUE THREAD

**Issue by ca20dnlm** [lukevella/Rallly#63]
or without deleting the participants?

**Comment by Empact** [comment]
Agree, it should not delete the votes for the remaining dates, when dates are removed. Alternatively, there should be a way to explicitly eliminate dates from consideration.

**Comment by dustyr** [comment]
I also agree.  Sometimes need to add a date or remove a date from consideration and it should not remove all other votes for other dates.

**Comment by lukevella** [comment]
I hear you. It's fixed in the new version. Unfortunately this was a limitation of the old version that I never found the time to fix.  Closing this as there's no further action needed right now.

**Comment by dustyr** [comment]
When will the new version be available?  Thanks.

**Comment by lukevella** [comment]
It's in beta right now so you can go ahead and use it already by signing up for early access. https://beta.rallly.co

In terms of the official release… I'm aiming for a mid-April.

**Comment by dustyr** [comment]
You probably already know this but there is a problem with selecting dates
in the beta.  It is off by 1 day.  For example, to select a Tuesday, you
must click on Wednesday.  Just wanted to let you know.

**Comment by lukevella** [comment]
Hi @dustyr, Yes I can see the issue is related to time zones. Going to try and get a fix out for this tomorrow.
